cypress/support/commands.js

```diff
@@ -147,3 +163,91 @@ Cypress.Commands.add(
     cy.wait('@createDetector');
   }
 );
+
+/**
+ *****************************
+ IM PLUGIN COMMANDS
```

Comment:
Same here cc @ohltyler

Reply:
Agreed. I think this can be done in a similar strategy to my [first comment](https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/94#discussion_r809456457). 

To get added in the cypress config properly, be sure to import your new commands file in [cypress/support/commands/index.js](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/support/index.js#L17) (ref: https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/94#discussion_r809456457)